CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 ## Unreleased
 
+## v0.10.0
+
+IMPROVEMENTS:
+
+* Update golang.org/x/text to v0.3.8 [[GH-48]](https://github.com/hashicorp/vault-plugin-secrets-openldap/pull/48)
+
```

Comment:
There was also the CreateOperation changes (https://github.com/hashicorp/vault-plugin-secrets-openldap/pull/50) in case we wanted to mention it here.

Reply:
Whoops! Thanks. Added in https://github.com/hashicorp/vault-plugin-secrets-openldap/pull/52/commits/8e9907e8fcd9961219db60dc03db1161884fa263.